lib/components/StatusTrend/StatusTrend.styled.ts

```diff
@@ -0,0 +1,37 @@
+import styled from "styled-components";
+
+import { ParagraphLarge } from "../Typography";
+import { StatusTrendProps } from "./StatusTrend.types";
+
+export const StatusTrendWrapper = styled.div<Pick<StatusTrendProps, "variant">>`
+  display: flex;
+  align-items: flex-start;
+  justify-content: center;
+  width: max-content;
+`;
+
+export const LabelWrapper = styled.div<{ hasSecondaryLabel: Boolean }>`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: flex-start;
+  margin-top: ${(props: any) => (props.hasSecondaryLabel ? "-7px" : "-4px")};
+  margin-left: ${({ theme }) => theme.spacings.sm};
+`;
+
+export const StyledPrimaryLabel = styled(ParagraphLarge)<
+  Pick<StatusTrendProps, "variant">
+>`
+  color: ${({ theme, variant }) => theme.statusTrend.labelColor[variant]};
+  font-size: 16px;
```

Comment:
don't we need to use the standard sizes?

Reply:
+1, we should.

Reply:
I will adjus that, no problem